doc/contributing/contribute_code.md

```diff
@@ -108,6 +112,51 @@ standard GitHub pull request model.
 Changes can be submitted to the project by creating a pull request on the 
 [z390 project repository](https://github.com/z390development/z390).
 
+## Creating a new release
+
+The z390 project uses a file-based versioning system with git tags. Version
```

Comment:
Are we supposed to create a new release/distro with every PR ??

Reply:
The branch name is just a suggestion. The name does not matter. The idea is that you need to create a branch, update version.txt and then create a pull request. You could include a version update with every PR if you wanted to.. But it will only release IF the version.txt is changed.